server/sequelize/sequelize.ts

```diff
@@ -0,0 +1,4 @@
+import {Sequelize} from 'sequelize-typescript';
+import {Configuration} from './sequelize.config';
+
+export const sequelize = new Sequelize(Configuration['development']);
```

Comment:
Don't force configuration - will not work for CI - hard to maintain

Reply:
What is the best way to switch between configurations depending on environment? process.env.NODE_ENV || 'development'?

Reply:
Yes. This is ok